tiledb/sm/tile/generic_tile_io.cc

```
@@ -58,6 +59,30 @@ GenericTileIO::GenericTileIO(ContextResources& resources, const URI& uri)
 /*               API              */
 /* ****************************** */
 
+tuple<Status, optional<Tile>> GenericTileIO::load(
```

Let's prefer to throw rather than return Status for new functions (and as much as feasible for refactoring).

---

I'll give it a whirl.

---

Fixed.